arm/Microsoft.Storage/storageAccounts/readme.md

```diff
@@ -53,6 +53,42 @@ The default parameter values are based on the needs of deploying a diagnostic st
 | `userAssignedIdentities` | object | `{object}` |  | Optional. Mandatory 'managedServiceIdentity' contains UserAssigned. The identy to assign to the resource. |
 | `vNetId` | string |  |  | Optional. Virtual Network Identifier used to create a service endpoint. |
 
+### Parameter Usage: `privateEndpoints`
```

Comment:
Double up. Remove one of the privateEndpoints

Reply:
done